src/lib/trajectory/parse.ts

```
@@ -836,498 +507,390 @@
     } catch {
-      throw new Error(
-        `Content is not valid JSON, XYZ trajectory, single XYZ, VASP XDATCAR, or HDF5 format`,
-      )
+      throw new Error(`Unsupported text format`)
     }
   }
 
-  if (!data || typeof data !== `object`) {
-    throw new Error(`Invalid trajectory data: must be an object or array`)
-  }
+  if (!data || typeof data !== `object`) throw new Error(`Invalid data format`)
```

Biome (1.9.4)

[error] Invalid `typeof` comparison value: this expression is not a string literal

not a string literal

(lint/suspicious/useValidTypeof)